src/Bridge/Doctrine/EventListener/PublishMercureUpdatesListener.php

```diff
@@ -190,6 +202,23 @@ private function storeObjectToPublish($object, string $property): void
             }
         }
 
+        if ($options['topics'] ?? false) {
+            $topics = [];
+            foreach ((array) $options['topics'] as $topic) {
+                if (\is_string($topic) && 0 === strpos($topic, '@=')) {
+                    if (null === $this->expressionLanguage) {
+                        throw new \LogicException('The "@=" expression syntax cannot be used without the Expression Language component. Try running "composer require symfony/expression-language".');
+                    }
+
+                    $topics[] = $this->expressionLanguage->evaluate(substr($topic, 2), ['object' => $object]);
+                } else {
```

Comment:
Nitpicking: use `continue` instead of the `else` to reduce the complexity (I would also invert the condition to transform it into a guard clause).